tests/python/ConfigTest.py

```diff
@@ -773,83 +775,287 @@ def test_virtual_display(self):
+        views = cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)
+        self.assertEqual(views.__len__(), 0)
+
+    def test_virtual_display_with_active_displays(self):
+        """
+        Test the virtual display instantiation when active displays
+        and views are defined.
+        """
+
+        self.cfg_active_display.validate()
+        displays = self.cfg_active_display.getDisplays()
+        self.assertEqual(displays.__len__(), 1)
+        views = self.cfg_active_display.getViews('sRGB')
+        self.assertEqual(views.__len__(), 1)
+
+    def test_virtual_display_v1(self):
+        """
+        Test that the virtual display is only supported by v2 or higher.
+        """
+
+        with self.assertRaises(OCIO.Exception):
+            cfg = OCIO.Config().CreateFromStream(SIMPLE_CONFIG_VIRTUAL_DISPLAY_V1)
+
+        cfg = OCIO.Config().CreateRaw()
+        cfg.addVirtualDisplaySharedView('sview')
+        cfg.setMajorVersion(1)
+        with self.assertRaises(OCIO.Exception):
+            cfg.validate()
+
+        with self.assertRaises(OCIO.Exception):
+            cfg2 = OCIO.Config().CreateFromStream(cfg.serialize())
+
+    def test_virtual_display_exceptions(self):
+        cfg = OCIO.Config().CreateFromStream(SIMPLE_CONFIG_VIRTUAL_DISPLAY_EXCEPTION)
+        cfg.validate()
+
+        # Test failures for shared views.
+        with self.assertRaises(OCIO.Exception) as cm:
+            cfg.addVirtualDisplaySharedView('sview1')
+        self.assertEqual(str(cm.exception),
+                         "Shared view could not be added to virtual_display: " +
+                         "There is already a shared view named 'sview1'.")
+
+        cfg.addVirtualDisplaySharedView('sview2')
+        with self.assertRaises(OCIO.Exception) as cm:
+            cfg.validate()
+        self.assertEqual(str(cm.exception), "Config failed validation. " +
+                         "The display 'virtual_display' contains a shared " +
+                         "view 'sview2' that is not defined.")
+
+        cfg.removeVirtualDisplayView('sview2')
+        cfg.validate()
+
+        # Test failures for views.
+        with self.assertRaises(OCIO.Exception) as cm:
+            cfg.addVirtualDisplayView('Raw', 'Film', 'raw')
+        self.assertEqual(str(cm.exception), "View could not be added to " +
+                         "virtual_display in config: View 'Raw' already exists.")
+        
+        cfg.addVirtualDisplayView('Raw1', 'Film', 'raw1')
+        with self.assertRaises(OCIO.Exception) as cm:
+            cfg.validate()
+        self.assertEqual(str(cm.exception), "Config failed validation. " +
+                         "Display 'virtual_display' has a " +
+                         "view 'Raw1' that refers to a color space" +
+                         " or a named transform, 'raw1', which is not defined.")
+        
+        cfg.removeVirtualDisplayView('Raw1')
+        cfg.validate()
+
+        cfg.addVirtualDisplayView('Raw1', 'raw', 'look')
```

Comment:
It should be  `cfg.addVirtualDisplayView('Raw1', 'Film', 'raw1', 'look')` i.e. look is the fourth argument.